Sources/LiveViewNativeAVKit/VideoPlayerView.swift

```diff
@@ -0,0 +1,232 @@
+    @_documentation(visibility: public)
+    #endif
+    @LiveBinding(attribute: "is-muted") private var isMuted: Bool
+
+    /// The current playback time of the video player in seconds.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    @LiveBinding(attribute: "playback-time") private var playbackTime: Float64
+
+    /// The interval at which the playback time is updated.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    @LiveBinding(attribute: "playback-time-update-interval") private var playbackTimeUpdateInterval: Float64
+
+    /// The current time control status of the video player (playing, paused, etc.).
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    @LiveBinding(attribute: "time-control-status") private var timeControlStatus: String
+
+    /// If true, the video will play when the view appears.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    var autoplay: Bool = false
+
+    /// The URL of the video to play.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    var url: URL
+
+    @LiveContext<R> private var context
+    @ObservedElement private var element: ElementNode
+    @StateObject private var observer: VideoPlayerObserver<R>
+
+    init(element: ElementNode) {
+        if let value = element.attributeValue(for: "autoplay") {
+            self.autoplay = Bool(value)!
+        }
+        if let value = element.attributeValue(for: "url") {
+            self.url = URL(string: value)!
+        } else {
+            self.url = URL(string: "")!
+        }
+        let interval = CMTime(seconds: 0.01, preferredTimescale: CMTimeScale(NSEC_PER_SEC))
+
+        _observer = StateObject(wrappedValue: VideoPlayerObserver(interval: interval))
+    }
+
+    var body: some View {
+        VideoPlayer(player: self.observer.player)
+            .onAppear {
+                timeControlStatus = "paused"
+                observer.player.replaceCurrentItem(with: AVPlayerItem(url: url))
+
+                if self.autoplay {
+                    self.observer.player.play()
+                }
+            }
+            .onReceive(context.coordinator.receiveEvent("pause"), perform: performPause)
+            .onReceive(context.coordinator.receiveEvent("play"), perform: performPlay)
+            .onReceive(context.coordinator.receiveEvent("seek"), perform: performSeek)
+            .onReceive(observer.player.publisher(for: \.isMuted)) { value in isMuted = value }
+            .onReceive(observer.player.publisher(for: \.timeControlStatus)) { value in syncTimeControlStatus(status: value) }
+            /// This is a workaround for SwiftUI not supporting `onReceive` for `AVPlayer`'s `currentTime` property.
+            .onReceive(observer.assigns.$playbackTime.throttle(for: AVKitHelpers.floatToStride(float: playbackTimeUpdateInterval), scheduler: RunLoop.current, latest: true)) { value in playbackTime = value }
```

Comment:
You should be able to do this:
```suggestion
            .onReceive(observer.assigns.$playbackTime.throttle(for: .init(playbackTimeUpdateInterval), scheduler: RunLoop.current, latest: true)) { value in playbackTime = value }
```